engine/src/main/java/org/terasology/logic/players/CameraClientSystem.java

```diff
@@ -43,19 +61,34 @@
     EntityManager entityManager;
 
     @ReceiveEvent
-    public void onAutoMountCamera(OnChangedComponent event, EntityRef client, AutoMountCameraComponent autoMountCameraComponent, ClientComponent clientComponent) {
-        if (clientComponent.camera.exists()) {
-            client.send(new ResetCameraEvent());
+    public void ensureCameraEntityCreatedOnChangedClientComponent(OnChangedComponent event, EntityRef client, ClientComponent clientComponent) {
+        if (localPlayer.getClientEntity().equals(client)) {
+            ensureCameraEntityCreated();
         }
     }
 
     @ReceiveEvent
-    public void ensureCameraEntityCreated(OnActivatedComponent event, EntityRef entityRef, ClientComponent clientComponent) {
-        if (!clientComponent.camera.exists()) {
+    public void ensureCameraEntityCreatedOnActivateClientComponent(OnActivatedComponent event, EntityRef client, ClientComponent clientComponent) {
```

Comment:
Did you mean "...OnActivate**d**ClientComponent"?

Also, part of me would like "...OnClientComponentActivation" better but I can see you are being consistent with the OnActivatedComponent class, which is perhaps the source of the problem...